---
layout: page
title: Research Projects
---
<div class=text-justify>
  
&ensp; Plant pathogens exert one of the strongest selective pressures on plant populations. This leads to progressive changes in the genetic makeup of both the pathogen as well the plant populations, which is usually referred to as an arms race. Fungal pathogens are one of the highly evolved groups of microorganisms that affect various plant species and strongly differ in important life history traits such as dispersal mechanisms, type of reproduction and modes of parasitism. Economic damage due to fungal diseases in crops is estimated to be over 200 billion USD.  
  
&ensp; Our lab focusses on some of the major fungal diseases of the *Brassicaceae* family, especially *Alternaria* Blight or *Alternaria* Leaf spot caused by *Alternaria brassicae*. No source of resistance is known in any of the cultivated species of *Brassicas*. Our aim is to decipher the genetic factors underlying the pathogenesis of *A. brassicae* in *Brassicas* especially *B. juncea* and use the knowledge gained therein to assist disease resistance breeding programmes. The research projects ongoing in the lab are as below.  

**1.	Prospecting for resistance genes in the wild germplasm of *Brassicas* and *Arabidopsis thaliana* accessions locally adapted to the India subcontinent**

<img src="/img/Prj1.jpg" align='left'>

Natural variation in *A. thaliana* has been successfully utilized by our lab earlier to identify multiple genes/loci involved in resistance to A. brassicae (Rajaramohan S *et al*., 2017; Rajaramohan S *et al*.,2018) using GWA and QTL mapping approaches. Characterization of these genes and QTLs is underway. We aim to further identify durable resistance genes from wild germplasm of *Brassicas* and local populations of *A. thaliana*. Gene identification in the wild species of *Brassicas* will be greatly aided by the increasing genomic information that is becoming available due to reducing sequencing costs. This project will involve screening of the wild germplasm and landraces for resistance to *A. brassicae* and identify potential pre-breeding materials.

**2.	Identification of *A. brassicae* effectors and elucidating their role in pathogenesis**

<img src="/img/Prj2.jpg" align='right'>

We have sequenced and produced a highly-contiguous assembly of the genome of *A. brassicae* using Nanopore minION sequencing. Using various annotation pipelines, we have been able to predict and annotate various CAZy (Carbohydrate active enzymes), effectors, and secondary metabolite clusters. This project involves the function characterization of these identified effectors and secondary metabolites. 

**3.	Exploring the genetic diversity of *A. brassicae* across India to gain insights into the evolutionary history of the pathogen vis-à-vis its many host species**

Understanding a pathogen’s genetic diversity is crucial to the effectiveness and durability of host resistance. Population genetics of plant pathogens has enormous potential to unravel the evolutionary forces acting on pathogen population, and this information can be used to develop strategies of resistance breeding. We aim to catalogue the genetic diversity of various isolates of *A. brassicae* across India at the genomic level and analyse their implications in pathogenesis.

</div>

